lib/tasks/mastodon.rake

```diff
@@ -521,12 +535,53 @@ namespace :mastodon do
 
           prompt.ok "You can login with the password: #{password}"
           prompt.warn 'You can change your password once you login.'
+        else
+          prompt.warn 'Okay.'
+        end
+
+        prompt.say "\n"
+
+        blocklist = prompt.select("Would you like to import an initial domain blocklist for your instance?\n") do |menu|
+          menu.enum '.'
+
+          domain_blocklists.each do |list|
+            menu.choice "#{list[:name]} - website: #{list[:website]}", { url: list[:csv], name: list[:name] }
+          end
+          menu.choice 'No, skip setting up an initial blocklist', nil
         end
+
+        prompt.say "\n"
+
+        if db_connection_works && blocklist.present?
+          prompt.say "Starting import of #{blocklist[:name]} blocklist...\n"
+
+          cmd_env = env.transform_values(&:to_s).merge({ 'RAILS_ENV' => 'production' })
+          cmd = "./bin/tootctl domains blocklist import \"#{blocklist[:name]}\" \"#{blocklist[:url]}\""
+
+          Open3.popen2e(cmd_env, cmd) do |_stdin, stdout_err, wait_thr|
+            while (line = stdout_err.gets)
+              prompt.say "> #{line.chomp}", color: :dim
+            end
+
+            exit_status = wait_thr.value
+            if exit_status.success?
+              prompt.say 'Successfully imported blocklist!'
+            else
+              prompt.error 'Import failed!'
+            end
+          end
+        else
+          prompt.say 'Okay! You can always download a domain blocklist CSV later and import it via the Admin panel'
+        end
+
+        prompt.say "\n"
+        prompt.ok 'All done. Bye!'
       else
+        prompt.say "\n"
         prompt.warn 'Nothing saved. Bye!'
       end
     rescue TTY::Reader::InputInterrupt
-      prompt.ok 'Aborting. Bye!'
+      prompt.ok "\nAborting. Bye!"
```

Comment:
This fixes an issue in the output where the abort line would be joined to the preceding question line.